www/.htaccess

```diff
@@ -18,29 +18,31 @@ RewriteEngine on
 #       pages to new pages.
 
 # 302 (temp):
-#       docs     -> docs/en/latest/
-#       docs/    -> docs/en/latest/
-#       docs/XX  -> docs/XX/latest/
-#       docs/XX/ -> docs/XX/latest/
+#       docs/          -> docs/[default language]/
+#       docs/XX        -> docs/XX/
+#       docs/XX/       -> docs/XX/latest
+#       docs/XX/latest -> docs/XX/latest/
 #
 # NOTE:
-#       these can both apply, turning "/docs" into "/docs/en/latest/"
-RewriteRule ^.*docs/$ {{site.baseurl}}/docs/en/
-RewriteRule ^.*docs/(\w\w)/$ {{site.baseurl}}/docs/$1/latest/ [R=302,L]
-
-# 302 (temp):
-#       docs/XX/latest -> docs/XX/latest/
-RewriteRule ^.*docs/(\w\w)/latest$ {{site.baseurl}}/docs/$1/latest/ [R=302,L]
+#       these can all apply, turning "/docs" into "/docs/[default language]/latest/"
+RewriteRule ^.*docs/$ {{site.baseurl}}/docs/{{site.language}}/
+RewriteRule ^.*docs/(\w\w(?:-\w\w)?)$ {{site.baseurl}}/docs/$1/
```

Comment:
`/docs/en/index.html` does not exist, right?

Looks like `/docs/en` will go through three redirects:
```
/docs/en -> /docs/en/ -> /docs/en/latest -> /docs/en/latest/
```

Can't all of those just be one step instead of chaining like that? Makes it hard to read.